phpdotnet/phd/Options/Handler.php

```diff
@@ -33,29 +33,31 @@ public function optionList(): array
             'quit'         => 'Q',         // Do not run the render. Use with -S to just save the config.
             'memoryindex'  => 'M',         // Use sqlite in memory rather then file
             'packagedir'   => 'k:',        // Include path for external packages
-        );
+        ];
     }
 
-    public function option_M($k, $v)
+    public function option_M(string $k, mixed $v): void
     {
         $this->option_memoryindex($k, $v);
     }
-    public function option_memoryindex($k, $v)
+
+    public function option_memoryindex(string $k, mixed $v): void
     {
         Config::set_memoryindex(true);
     }
 
-    public function option_f($k, $v)
+    public function option_f(string $k, mixed $v): void
     {
-        if ($k == "f") {
-            return $this->option_format($k, $v);
-        } else {
-            return $this->option_outputfilename($k, $v);
+        if ($k === "f") {
```

Comment:
Might make sense to have the `"f"` be a class constant (maybe protected/private?)

Reply:
I'd like to tackle these strings in a follow up PR as I think the mapping of short option, long option and handler method should be in some sort of single data structure that is more straightforward to work with (array of value objects maybe?). But this needs some more thought put into it and I'm not sure what would be the best way to go about this.